vochain/scrutinizer/process.go

```diff
@@ -114,20 +134,34 @@ func (s *Scrutinizer) ProcessList(entityID []byte, namespace uint32,
 }
 
 // ProcessCount return the number of processes indexed
-func (s *Scrutinizer) ProcessCount() int64 {
-	c, err := s.db.Count(&Process{}, nil)
+func (s *Scrutinizer) ProcessCount(entityID []byte) int64 {
+	var c int
+	var err error
+	if len(entityID) == 0 {
+		c, err = s.db.Count(&Process{}, nil)
+	} else {
+		c, err = s.db.Count(&Process{}, badgerhold.Where("EntityID").Eq(entityID))
+	}
 	if err != nil {
 		log.Warnf("cannot count processes: %v", err)
 	}
 	return int64(c)
 }
 
 // EntityList returns the list of entities indexed by the scrutinizer
-func (s *Scrutinizer) EntityList(max, from int) []string {
+func (s *Scrutinizer) EntityList(searchTerm string, max, from int) []string {
```

Comment:
Make searchTerm the last value since it is "optional". Also please, specify it can be empty on the method description